CSLabs.Api/Models/Maintenance.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CSLabs.Api.Models
+{
+    public class Maintenance
+    {
+        public int Id { get; set; }
+        public bool IsMaintenanceMode { get; set; }
+        public bool IsRestorationTimeKnown { get; set; }
+        public int RestorationTime { get; set; }
```

Comment:
Shouldn't a maintenance have a start time?

Reply:
@Cliftonz Can you verify my comments on this PR to make sure that's what you want?

Reply:
yes they should have a start time.